tools/autograd/derivatives.yaml

```diff
@@ -401,15 +416,18 @@
 - name: cross(Tensor self, Tensor other, int? dim=None) -> Tensor
   self: other.conj().cross(grad, dim)
   other: grad.cross(self.conj(), dim)
+  result: "at::cross(self_t, other_p, dim.has_value() ? *dim : -1) + at::cross(self_p, other_t, dim.has_value() ? *dim : -1)"
```

Comment:
If dim is None... doesn't dim default "to the first dimension found with size 3?", not -1? https://pytorch.org/docs/stable/generated/torch.cross.html

Reply:
Yes I should just pass "dim" here.